Sources/LiveViewNative/Stylesheets/ParseableTypes/Styles/AnyTabViewStyle+ParseableModifierValue.swift

```diff
@@ -28,6 +28,8 @@ enum AnyTabViewStyle: ParseableModifierValue {
                 #if os(watchOS)
                 if #available(watchOS 10.0, *) {
                     ConstantAtomLiteral("verticalPage").map({ Self.verticalPage(transitionStyle: nil) })
+                }
+                if #available(watchOS 10.0, *) {
                     VerticalPage.parser(in: context).map({ Self.verticalPage(transitionStyle: $0.transitionStyle) })
                 }
```

Comment:
Should this conditional just be combined with the one before it?

Reply:
This actually needed to be restructured because in that state it caused a warning about `buildLimitedAvailability` not be being available.